pkg/sensors/accelerator/devices/device.go

```diff
@@ -121,8 +121,13 @@ func (r *Registry) MustRegister(a string, d DeviceType, deviceStartup deviceStar
 		return
 	}
 	klog.V(5).Infof("Adding the device to the registry [%s][%s]", a, d.String())
-	r.Registry[a] = map[DeviceType]deviceStartupFunc{
-		d: deviceStartup,
+	m, ok := r.Registry[a]
+	if !ok {
+		r.Registry[a] = map[DeviceType]deviceStartupFunc{
+			d: deviceStartup,
+		}
+	} else {
+		m[d] = deviceStartup
```

Comment:
shouldn't we panic here instead? The accelerator has ready been defined and there is now an attempt to overwrite it. IMHO, we should panic and aligns with the `Must` in MustRegister.